lib/langchain/llm/response/openai.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module Langchain::LLM::Response
+  class OpenAI < Base
+    attr_reader :raw_response
+
+    def model
+      raw_response["model"]
+    end
+
+    def type
```

Comment:
I'm wondering if we should just fully simplify this and get this of the `value()` method and the needed `type()`.